packages/ui/src/proposals/modals/AddNewProposal/machine.ts

```diff
@@ -637,6 +646,15 @@ export const addNewProposalMachine = createMachine<AddNewProposalContext, AddNew
             },
           },
         },
+        setMembershipLeadInvitationQuota: {
+          on: {
+            SET_AMOUNT: {
+              actions: assign({
+                specifics: (context, event) => ({ ...context.specifics, amount: (event as SetAmountEvent).amount }),
```

Comment:
```suggestion
                specifics: (context, event) => ({ ...context.specifics, amount: event.amount }),
```

Could you check if the `as` is necessary? It should be inferred from the events